development-defaults.ini

```diff
@@ -113,16 +149,8 @@ no_shirt     = 0
 'Squarewave Lanyard'          = 3
 
 
-
 [enums]
 
-[[badge_status]]
```

Comment:
Sure, that seems easy enough to add back in.  It sounds like it would combine nicely with the notion of "archiving" an attendee, which is what we should do instead of deleting them completely.  When someone is archived and removed from the current year's set of attendees, we could specify the reason.  We could also set it on active registrations as well as desired.